blackbox/config.py

```diff
@@ -43,21 +43,23 @@ class Prefixes(metaclass=YAMLGetter):
     @classmethod
     def parse_config(cls, config_path: Path = None):
         """Parse the config from the blackbox.yaml file."""
+
         # If config_path is passed, use that.
+        env_config_path = os.environ.get("BLACKBOX_CONFIG_PATH")
+
+        # If there is a path to the config, parse it
```

Comment:
I'm thinking the comment on line 47 can just be above `if config_path`, where it seems to belong.

The environment get doesn't need a block comment.